tutordistro/patches/openedx-dockerfile-post-python-requirements

```diff
@@ -1,4 +1,4 @@
-{%- if DISTRO_INSTALL_EDNX_REQUIREMENTS %}
+{%- if DISTRO_INSTALL_EDNX_REQUIREMENTS is defined and DISTRO_INSTALL_EDNX_REQUIREMENTS %}
```

Comment:
Maybe we could remove this and add a setting like 


```yaml
INSTALL_EXTRA_FILE_REQUIREMENTS:
    - ./requirements/edunext/base.txt
    - ./requirements/nelp/test.txt
```

This is more generic maybe? Remember that these changes are for remove all "mandatory distro defaults"

Reply:
Yes, you're right. I will add the setting.